docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -43,7 +43,7 @@ For `date-fns`, import the locale and pass it to `LocalizationProvider`:
 :::info
 Both `date-fns` major versions (v2.x and v3.x) are supported.
 
-A single adapter can not work for both `date-fns` v2.x and v3.x, because the way functions are exported has been changed in v3.x.
+A single adapter cannot work for both `date-fns` v2.x and v3.x, because the way functions are exported has been changed in v3.x.
```

Comment:
https://www.merriam-webster.com/grammar/cannot-vs-can-not-is-there-a-difference#:~:text=Both%20cannot%20and%20can%20not,general%2C%20it%20is%20somewhat%20informal.

Reply:
I'll make a PR to remove all the "can't" and "can not" from our doc :laughing: